Sources/GryphonLib/TranspilationPass.swift

```diff
@@ -4712,6 +4712,20 @@ public class FixExtensionGenericsTranspilationPass: TranspilationPass {
 	}
 }
 
+public class EscapeDollarSignsInStringsTranspilationPass: TranspilationPass {
+    // gryphon insert: constructor(ast: GryphonAST, context: TranspilationContext):
+    // gryphon insert:     super(ast, context) { }
+    
+    override func replaceLiteralStringExpression( // gryphon annotation: override
+        _ literalStringExpression: LiteralStringExpression) -> Expression {
+        let replacedLiteralStringExpression = LiteralStringExpression(
+            range: literalStringExpression.range,
+            value: literalStringExpression.value.replacingOccurrences(of: "$", with: "\\$"),
+            isMultiline: literalStringExpression.isMultiline)
+        return super.replaceLiteralStringExpression(replacedLiteralStringExpression)
+    }
+}
+
 /// Kotlin initializers cannot be marked as `open`.
```

Comment:
I think you forgot to add a call to the `EscapeDollarSignsInStringsTranspilationPass` class at the end of the file, around line 4842